etc/kernel-launchers/scala/toree-launcher/src/main/scala/launcher/ToreeLauncher.scala

```diff
@@ -175,26 +184,27 @@ object ToreeLauncher {
     initArguments(args)
 
     if (profilePath == null && kernelId == null){
-      println("At least one of '--profile' or '--RemoteProcessProxy.kernel_id' must be provided - exiting!")
+      logger.info("At least one of '--profile' or '--RemoteProcessProxy.kernel_id' " +
+        "must be provided - exiting!")
       sys.exit(-1)
     }
     if (kernelId == null){
-      println("WARNING: Parameter 'connection_file' is deprecated.  Update kernel.json file to use " +
-            "'--RemoteProcessProxy.kernel-id {kernel_id}'")
+      logger.info("WARNING: Parameter 'connection_file' is deprecated.  " +
```

Comment:
Should probably use `logger.warn`.  We could also remove the `WARNING:` prefix as well.